app/services/kredits.js

```diff
@@ -46,18 +46,33 @@ export default Service.extend({
   }),
 
   kreditsByContributor: computed('contributionsUnconfirmed.[]', 'contributors', function() {
```

Comment:
I think you might want to change the dependency, so the computed property updates when the `vetoed` status of any of the contributions changes.

```suggestion
  kreditsByContributor: computed('contributionsUnconfirmed.@each.vetoed', 'contributors', function() {
```

Reply:
Cool. Will that still account for new objects in the collection, and changes of other properties?

Reply:
Yes, it will also update when the array itself changes (e.g. elements added or removed). If you want to add additional properties as dependencies, you would add them like this `contributionsUnconfirmed.@each.{vetoed,otherProperty}`.